ssc/cmod_battery_eqns.h

```diff
@@ -69,6 +69,53 @@ static const char* calculate_thermal_params_doc =
 
 SSCEXPORT bool Calculate_thermal_params(ssc_data_t data);
 
+static const char* Reopt_size_standalone_battery_params_doc =
+"Given battery costs and a load, get the optimal PV and battery sizes. Wind is disabled.\\n"
+"Maps SAM compute module inputs to those of the ReOpt API:\\n"
+"Battery technology paired with Residential, Commercial, Third Party or Host Developer financing.\\n\\n"
+"Optional: if missing, variable will be set to default value if documented, to REopt defaults otherwise.\\n"
+"Conditional: only required if the variable it's meant to replace is missing.\\n"
+"REopt's 'microgrid_upgrade_cost_pct' set to 0 because SAM doesn't apply this cost multiplier.\\n"
+"REopt inputs not used in this function will be REopt's defaults.\\n"
+"Some additional REopt inputs are required, such as lon and lat. See https://nrel.github.io/REopt.jl/dev/reopt/inputs/ for a full set of inputs\\n\\n"
```

Comment:
Is lat and lon still required?

Reply:
I don't see lat and lon in the new battery eqn being passed in?

Reply:
The battery compute module does not have lat and lon, they need to be added to the post after this function call.

Reply:
I see that it's added in the pv equation. So is this standalone battery post a viable submission to REopt in itself?

Reply:
It is not. This line is intended to be a hint that additional inputs are required, is there a better way to phrase it?

Reply:
"The post produced here is not a complete submission. Further additions are required, such as x and y, performed by function z"?

In that case, what is the intended use case in adding it to PySAM's standalone battery module? The user then adds the missing variables?